packages/core/src/external/carequality/command/documents/send-ihe-gateway-results.ts

```diff
@@ -93,47 +103,50 @@ async function checkNumberOfResults(
   return "";
 }
 
-async function isDQComplete(
+async function isIheGatewayResultsComplete(
   sequelize: Sequelize,
+  resultsTable: string,
   requestId: string,
-  numOfLinksInRequest: number
+  numOfGatewaysInRequest: number
 ): Promise<boolean> {
-  const dqResultCount = await getDocumentQueryResultCount(sequelize, requestId);
-  return dqResultCount >= numOfLinksInRequest;
+  const iheResultCount = await getIHEGatewayResultCount(sequelize, resultsTable, requestId);
+  return iheResultCount >= numOfGatewaysInRequest;
 }
 
-async function getDocumentQueryResultCount(
+async function getIHEGatewayResultCount(
   sequelize: Sequelize,
+  resultsTable: string,
   requestId: string
 ): Promise<number> {
   try {
-    const query = `SELECT COUNT(*) FROM ${TABLE_NAME} WHERE ${REQUEST_ID_COLUMN} = '${requestId}';`;
+    const query = `SELECT COUNT(*) FROM ${resultsTable} WHERE ${REQUEST_ID_COLUMN} = '${requestId}';`;
```

Comment:
nit: shouldn't we use Sequelize models to query things like this? i.e. 

```
return await xxx.count({
    where: { requestId },
  });
```

Reply:
This is being used in the lambda so there is no access to the models

Reply:
How many records are we expecting on this table? AFAIK there's no index on `requestId`

Reply:
We should expect it to grow over time as its going to be the results of each gateway for each patient. Ill add the index in a migration